cli/lib/tasks/download.js

```diff
@@ -54,6 +54,20 @@ const getBaseUrl = () => {
   return defaultBaseUrl
 }
 
+const getCA = () => {
+  let ca
+
+  if (util.getEnv('CYPRESS_DOWNLOAD_MIRROR_CA')) {
+    ca = [util.getEnv('CYPRESS_DOWNLOAD_MIRROR_CA')]
```

Comment:
So this is the use case of embedding the entire CA cert in an environment variable? Is that common?

Reply:
I have followed `npm` config pattern. It accepts either `cafile` or `ca`.

https://docs.npmjs.com/cli/v7/using-npm/config#ca
https://docs.npmjs.com/cli/v7/using-npm/config#cafile

In previous [comment](https://github.com/cypress-io/cypress/pull/8877#issuecomment-721598739) I have explained why using standard `ca` and `cafile` config values from `npm` could potentially be a breaking change. Of course, I am happy to change back to `ca` and `cafile` if you think that following `npm` standard with slight risk of breaking change is better option.